source/_docs/configuration/basic.markdown

```diff
@@ -105,6 +106,15 @@ legacy_templates:
   required: false
   type: boolean
   default: false
+language:
+  description: "Default language used by Home Assistant. This may for example influence the language used by voice assistants. The language should be specified as an RFC 5646 language tag, and must be a language which Home Assistant is translated to."
```

Comment:
```suggestion
  description: "Default language used by Home Assistant. This may, for example, influence the language used by voice assistants. The language should be specified as an RFC 5646 language tag, and must be a language to which Home Assistant is translated to."
```